shared/src/main/scala/effekt/symbols/symbols.scala

```diff
@@ -233,16 +256,20 @@ package object symbols {
     val ret = Some(Effectful(tpe, Pure))
   }
 
-  sealed trait Effect extends TypeSymbol {
+  /** Effects */
+
+  sealed trait Effect {
+    def name: Name
+    def builtin: Boolean
     // invariant: no EffectAlias in this list
     def dealias: List[Effect] = List(this)
   }
 
-  case class EffectAlias(name: Name, effs: Effects) extends Effect {
+  case class EffectAlias(name: Name, effs: Effects) extends Effect with TypeSymbol {
```

Comment:
Why is `EffectAlias` and `UserEffect` a `TypeSymbol` but `Effect` isn't? This raises two questions in me:

1. Is there a subclass/-type of `Effect` which isn't a `TypeSymbol`?
2. Is an `EffectAlias` more than just a "pointer" to the original `Effect`?

Reply:
ha, tricky one. I do not want to make `Effect` a symbol since in the future there will be effects that are not symbols. For instance `State[Int]` is an effect, but not a symbol (`State` is, and `Int` is, but the application `State[Int]` is not a symbol).